ext/standard/password.c

```diff
@@ -167,6 +196,22 @@ PHP_FUNCTION(password_get_info)
 				add_assoc_long(&options, "cost", cost);
 			}
 			break;
+#if HAVE_ARGON2LIB
+		case PHP_PASSWORD_ARGON2I:
+		case PHP_PASSWORD_ARGON2D:
+			{
+				zend_long v = 0;
+				zend_long m_cost = PHP_PASSWORD_ARGON2_MEMORY_COST;
+				zend_long t_cost = PHP_PASSWORD_ARGON2_TIME_COST;
+				zend_long threads = PHP_PASSWORD_ARGON2_THREADS;
+
+				sscanf(hash, "$%*[argon2id]$v=" ZEND_LONG_FMT "$m=" ZEND_LONG_FMT ",t=" ZEND_LONG_FMT ",p=" ZEND_LONG_FMT, &v, &m_cost, &t_cost, &threads);
```

Comment:
TIL about `%[...]` sscanf format. Is this universally supported or an extension?